osmaxx/excerptexport/templates/excerptexport/exports_list.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'excerptexport/base.html' %}{% load i18n capture_as email_bodies static %}
+{% block breadcrumb %}
+    {{ block.super }}
+    <li class="active">{% trans 'My Exports' %}</li>
+{% endblock %}
+
+
+{% block main %}
+    <div class="container-row content">
+        <div class="col-lg-12">
+            <h2>My Exports</h2>
+            {% regroup exports by extraction_order.excerpt as grouped_export_list %}
+            {% for export_group in grouped_export_list %}
+                <div class="row">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h3 class="panel-title">{{ export_group.grouper.name }}</h3>
+                        </div>
+                        <div class="panel-body">
+                            <div class="col-lg-6">
+                                {% for export in export_group.list|slice:":5" %}
+                                    {% include 'excerptexport/partials/export_display.html' %}
+                                {% endfor %}
+{#                            TODO: fill in the url when the detail view is available #}
+                                {% if export_group.list|length > 5 %}<a href="">{% trans 'see full history' %}</a>{% endif %}
+                            </div>
+                            <div class="col-lg-6">
+                                {% if export_group.grouper.country %}
+                                    <div class="minimap" id="minimap-country-{{ export_group.grouper.id }}">Show the map! ID: country-{{ export_group.grouper.id }}</div>
+                                {% else %}
+                                    <div class="minimap" id="minimap-{{ export_group.grouper.id }}">Show the map! ID: {{ export_group.grouper.id }}</div>
+                                {% endif %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            {% empty %}
```

Comment:
:smile: Ah, cool!

I had already forgotten about [`for`...`empty`](https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#for-empty). (Or maybe I was never really aware of it in the first place.)